crrExistingObjects.js

```diff
@@ -94,12 +95,12 @@ function _markObjectPending(bucket, key, versionId, storageClass,
             VersionId: versionId,
         }, log, next),
         (mdRes, next) => {
-            objMD = mdRes;
+            objMD = new ObjectMD(mdRes);
             if (!_objectShouldBeUpdated(objMD)) {
                 skip = true;
                 return next();
             }
-            if (objMD.versionId) {
+            if (objMD.getVersionId()) {
```

Comment:
Further follow up on the other PR, the original `versionId` would be used if `getVersionId` was instead a get method.